3주차/BE/yesjjin99/src/entities/post.entity.ts

```diff
@@ -0,0 +1,17 @@
+import { BaseEntity, Entity, PrimaryGeneratedColumn, Column } from 'typeorm';
+
+@Entity()
+export class Posts extends BaseEntity {
```

Comment:
```suggestion
export class Posts{
```
https://typeorm.io/#/active-record-data-mapper
을 보시면 아시겠지만  BaseEntity는 Active Record pattern을 위해서 사용합니다.
일반적으로는 상속하지 않아도 문제 없습니다.